taxonomies/packaging_shapes.txt

```diff
@@ -277,9 +285,111 @@ fr:Batonnet, bâtonnets
 hu:rúd, rudak
 pt:Paus, pauzinhos, varetas de madeira
 
+fr:Stick
+# different - do not merge
+
 en:label, labels
 de:Etikett, Etiketten
 fr:étiquette, étiquettes
 hu:címke, címkék
 pt:Etiqueta, etiquetas, rótulo, rótulos
 weight:en:light
+
+en:Applicator
+fr:Applicateur
+#suggest:packaging_materials:en: en:plastic, en:cardboard
+
+fr:Bac
+
+en:Bag-in-box
+fr:Bag-in-box, Cubitainer
+
+en:strip, band
+fr:Bandelette
+#suggest:packaging_materials:en: en:plastic, en:metal, en:carboard, en:other
+
+en:Ampoule, ampul, ampule
+fr:Ampoule
+#suggest:packaging_materials:en: en:glass
+
+fr:Bidon
+
+en:Blister
+fr:Blister
+
+fr:Rouleau
+
+fr:Bobine
+
+fr:Caisse
+
+fr:Coupelle
+
+fr:Bombonne
+
+fr:Collerette
+
+fr:Muselet, Muselet et plaque
+#goes_with:en:glass bottle
+
+en:hanger
+fr:Cintre
+
+en:Wedge, corner piece
+fr:Cale, Coin, Cornière
+#suggest:packaging_materials:en: en:plastic, en:cardboard, en:metal, en:other
+
+# close to bottle, but also ampoule is translated to vial by citeo
```

Comment:
So we have English translations for all Citeo elements? Then can we just add them to all the new fr: only entries?